src/ts/client/utils.ts

```diff
@@ -45,5 +45,19 @@ export async function hashString(str: string): Promise<string> {
 
     return hashHex
   }
-  else {return 'local'} 
+  else {return 'local'}
+}
+
+/**
+ * Select the <body> element in the editor.
+ * @param editor The GrapesJS editor.
+ */
+
+export function selectBody(editor: Editor): void {
+  editor.select(editor.DomComponents.getWrapper())
+}
+
+export function isTextOrInputField(element: HTMLElement): boolean {
+  return element.tagName === 'TEXTAREA' ||
```

Comment:
you probably want to include the select? maybe other form elements?

Reply:
Yes, good idea!